ckanext/useraffiliation/templates/user/read_base.html

```diff
@@ -0,0 +1,7 @@
+{% ckan_extends %}
+
+{% block user_heading %}
+    <h1 class="heading">{{ user.display_name }}</h1>
+    <h2 class="heading small">{{ user.job_title }}, {{ user.affiliation }}</h2>
+{% endblock %}
```

Comment:
Interesting you have added this to the heading. Yes I think it could work.  Do you have a screen shot of what it actually looks like? I think the other option would be to extend `{% block user_info %}` instead?

Reply:
![image](https://user-images.githubusercontent.com/7795189/185364699-06cbcccd-48f5-4995-93db-75c0fd1684fe.png)

Reply:
I like it.  What happens if you have a long job title and/or affiliation? i.e. multiple lines.  Do you chop off, or wrap the ext?  If you wrap the text, does it still look professional and sensible.  My guess is that we will want to use something other than H2 formatting.  e.g. <div class="affiliation"> with
```
.affiliation{
   font-weight: bold;
   size: 1.2em;
   margin: 10px 0px;
}
```